### ISSUE TITLE
Move Documentation from ASP.NET Identity repo to Docs

### ISSUE THREAD

**Issue by toiyabe** [dotnet/AspNetCore.Docs#26429]
Recently I started a [discussion](https://github.com/dotnet/aspnetcore/issues/42663) and a PR to update a section of the [README.md](https://github.com/dotnet/AspNetCore/tree/main/src/Identity#identity) titled: "[Community Maintained Store Providers](https://github.com/dotnet/AspNetCore/tree/main/src/Identity#community-maintained-store-providers)."

During the discussion, it was suggested that the "store providers" discussion be moved from the source code README to the documentation site.

What are people's thoughts on this? Should this section be moved, and if so, where would this section be moved to?

**Comment by guardrex** [comment]
Hello @toiyabe ... Custom storage providers (building one) has a doc at ...

**Custom storage providers for ASP.NET Core Identity**
https://docs.microsoft.com/aspnet/core/security/authentication/identity-custom-storage-providers

Perhaps, a section could go there. Alternatively ... since that one is focused on *building a custom provider* ... there's another topic that might be considered for it at ...

**Community OSS authentication options for ASP.NET Core**
https://docs.microsoft.com/aspnet/core/security/authentication/community

**Comment by guardrex** [comment]
Yes, I think that's right. One has to interpret "options" *broadly* for that 2nd one to make sense. However, that could change. @Rick-Anderson might decide to keep all of the 3rd party/OSS stuff in one spot ... in that topic ... if so, that topic could be re-titled easily enough when the new section is added. Rick will be along in a bit to see what's what.

**Comment by toiyabe** [comment]
Hello @Rick-Anderson , I see you added "needs-more-info". Can I help answer something?

**Comment by Rick-Anderson** [comment]
We'll take a look at this after the .NET 7 wave.

**Comment by toiyabe** [comment]
Ok, I'll close this for now.

**Comment by Rick-Anderson** [comment]
@toiyabe  let's keep it open. Sounds like a good idea when we get caught up.

**Comment by Rick-Anderson** [comment]
@toiyabe We have [`is:open label:"code help wanted" ` issues](https://github.com/dotnet/AspNetCore.Docs/labels/code%20help%20wanted) for .NET 7 code samples help need if you're interested.

**Comment by toiyabe** [comment]
Thanks Rick! I will take a look.

**Comment by Rick-Anderson** [comment]
@toiyabe  I'll be creating more code help issues so check weekly if you have time.

**Comment by toiyabe** [comment]
Will do!
